backend-api/src/functions/adapters/aws/kms/types.ts

```diff
@@ -0,0 +1,8 @@
+export interface CredentialJwt {
```

Comment:
Nitpick - as this isn't itself a Jwt
```suggestion
export interface CredentialJwtPayload {
```

Reply:
Updated and the func name